content/en/docs/tasks/observability/logs/collecting-logs/index.md

```diff
@@ -86,7 +86,7 @@ The `kind: instance` stanza of configuration defines a schema for generated log
 to generate log entries for requests based on the attributes reported by Envoy.
 
 The `severity` parameter is used to indicate the log level for any generated
-`logentry`. In this example, a literal value of `"warning"` is used. This value will
+`logentry`. In this example, a literal value of `"warn"` is used. This value will
```

Comment:
I'm not sure this change is correct. Isn't this referring to the value in https://raw.githubusercontent.com/istio/istio/master/samples/bookinfo/telemetry/log-entry-crd.yaml?

Reply:
@douglas-reid PATL

Reply:
Yeah, I feel like, if anything, the sample output needs to be updated, not the text here.

Can you run through the task and confirm and put in an updated output to match (or, show the output and confirm that the sample yaml and this text should be updated) ?

Reply:
Okey，I'll give it a try.